client/post-editor/editor-confirmation-sidebar/index.jsx

```diff
@@ -41,6 +41,20 @@ class EditorConfirmationSidebar extends Component {
 		status: PropTypes.string,
 	};
 
+	state = {
+		hasUnclosed: false,
```

Comment:
I think that `hasOpened` is a better and less eyebrow-raising name here, althought it's not rigorously correct. There are two states that cause the sidebar to be rendered: `opened` and `publishing`, but that's a tiny detail.

The `publishing` state is not even used now: the sidebar closes immediately after clicking "Publish" and the "Publishing..." UI is rendered when the regular sidebar is visible again. The code shows signs of a feature that once worked and then got broken. Or a feature that was removed, but not completely.

Reply:
I might propose `isDeferringRender` or `hasRendered` or `isDormant` or `doFullRender`

Reply:
Yeah, I didn't really like the name too much. This is more sensible :)